src/app/app.component.ts

```diff
@@ -45,7 +46,8 @@ export class AppComponent implements OnInit, OnDestroy {
     private store: Store<any>,
     private router: Router,
     private electronService: ElectronService,
-    private gcsService: GcsService
+    private gcsService: GcsService,
+    private loginOut: LoginApiService,
```

Comment:
Maybe `loginService` for fields/variables would be better here, to follow the existing conventio.